mewarpx/mewarpx/diags_store/particle_diagnostic.py

```diff
@@ -0,0 +1,154 @@
+"""Diagnostic code that wraps the picmi.ParticleDiagonstics class"""
+
+import os
+import yt
+import glob
+import logging
+
+import matplotlib.pyplot as plt
+from mewarpx.diags_store.diag_base import WarpXDiagnostic
+
+from pywarpx import callbacks, picmi
+
+from mewarpx.mwxrun import mwxrun
```

Comment:
Go from built-in -> 3rd-party -> local
```suggestion
import os
import glob
import logging
import warnings

import matplotlib.pyplot as plt
import yt

from pywarpx import callbacks, picmi

from mewarpx.mwxrun import mwxrun
from mewarpx.diags_store.diag_base import WarpXDiagnostic
```